proposals/0095-AOA-multiplexing.md

```diff
@@ -14,161 +14,154 @@ This proposal aims to add AOA multiplexing capability to SdlRouterService.
 
 Currently SdlRouterService works for MultiplexTransport, which supports Bluetooth transport as the underlying transport. This proposal extends SdlRouterService to support both Bluetooth and AOA (Android Open Accessory) transports, so that multiple SDL applications can share the AOA transport without having to worry about USB connection.
 AOA protocol basically allows only a single application to access the USB accessory -- if multiple applications are designed to use the same accessory, the Android system asks which application should use the accessory and shows a permission dialog. This proposal allows multiple SDL applications to share the USB accessory.
+In addition, this proposal addresses extension of SdlRouterService to support simultaneous multiple transports, and AOA multiplexing will work as a part of simultaneous multiple transports. This proposal mainly assumes the scenario where utilizing Bluetooth multiplexing transport for RPC service, and utilizing AOA multiplexing transport for audio/video services.
+As you see the ![0141-multiple-transports proposal](0141-multiple-transports.md), the secondary transport will be specified by SDLCore, however.
```

Comment:
Please update to "As you see in the [0141-multiple-transports proposal](0141-multiple-transports.md),"